apps/condo/k6/src/ticket.test.ts

```diff
@@ -156,10 +185,22 @@ export async function checkFrontend (data) {
     const context = browser.newContext()
     const page = context.newPage()
 
+    const organizationForCheckFrontend = data.organizationForCheckFrontend
+    let cookie
+    let organizationLinkId
+
+    if (organizationForCheckFrontend.cookie && organizationForCheckFrontend.organizationLinkId) {
+        cookie = organizationForCheckFrontend.cookie
+        organizationLinkId = organizationForCheckFrontend.organizationLinkId
+    } else {
```

Comment:
Why do you need extra token for that organization? Is there a "special" user?

Reply:
It's a employee user with `property` ticket visibility type